README.aspnet.md

```diff
@@ -0,0 +1,103 @@
+﻿# Featured Tags
+
+* `4.8`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:4.8`
+* `3.5`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:3.5`
+
+# About This Image
+
+ASP.NET is a high productivity framework for building Web Applications using Web Forms, MVC, Web API and SignalR.
+
+This repository contains `Dockerfile` definitions for ASP.NET Docker images. 
```

Comment:
This line should be removed.  It is not correct when the readme is published to Docker Hub.